local/compose/build.go

```diff
@@ -36,6 +35,8 @@ import (
 	"github.com/docker/buildx/driver"
 	_ "github.com/docker/buildx/driver/docker" // required to get default driver registered
 	"github.com/docker/buildx/util/progress"
+	. "github.com/docker/cli/cli/config"
```

Comment:
Not sure what we've done elsewhere but do we want to load this into the current namespace? It might be cleaner to import this as cliconfig
```suggestion
	cliconfig "github.com/docker/cli/cli/config"
```